libs/app-theme/models/src/lib/components/icon.ts

```diff
@@ -25,6 +25,7 @@ export enum IconType {
   Search = "search",
   Settings = "settings",
   Spinner = "spinner",
+  SpinnerDark = "spinner-dark",
```

Comment:
No need for this. The spinner color can be changed by it's parent by simply defining the CSS' `color` property.